### DISCUSSION TITLE
LS9935BS+LS9736CP combined driver

### DISCUSSION THREAD

**Discussion by Kingsoun** [mrcodetastic/ESP32-HUB75-MatrixPanel-DMA#281] [Ideas]
There are a lot of LS9935BS + LS9736CP combination, it is recommended to increase these drivers, thank you！

**Comment by DarrylStrong** [comment]
Do you have links to any of these modules?

**Reply by Kingsoun** [reply]
Yes!   https://www.ledsic.com.cn/product/1-5
https://www.ledsic.com.cn/product/2-2
https://www.ledsic.com.cn/resolve

**Comment by DarrylStrong** [comment]
I did mean the complete PCB module rather that the ICs but looking at the datasheets the led driver looks to be a self scanning 16 channel driver.  As does the row driver look like a self scanning device. I suspect that this type of driver will not require the fantastic DMA capbilities of this library but will need I2C drivers instead. The Chinese datasheets are not very forthcoming!

**Reply by Kingsoun** [reply]
I’m sorry. I misspoke. It’s ICS. My understanding is that ICS and LED screens together form an LED module. As far as I know, this is a new high refresh LED scheme.

**Comment by board707** [comment]
[Kingsoun](https://github.com/Kingsoun)
There are a lot of such drivers now.
Generally they are metioned as S-PWM

**Reply by Kingsoun** [reply]
Esp32 if there is any way to use this S-PWM method of operation!
